src/lib/sslcrypto/openssl/library.py

```diff
@@ -0,0 +1,93 @@
+import os
+import sys
+import ctypes
+import ctypes.util
+
+
+# Disable false-positive _MEIPASS
+# pylint: disable=no-member,protected-access
+
+# Discover OpenSSL library
+def discover_paths():
```

Comment:
Recently there is new cmd line option for openssl lib
https://github.com/HelloZeroNet/ZeroNet/commit/1cc0ec3f31b3b8db4a534ec64ebe82629b81fa2e
i don't see any usage of this in this file change, i think it should also be considered while checking for openssl lib,

Reply:
IIRC ZeroNet redefines find_library so it should work. It's worth testing though.

Reply:
Should work now.